app/views/temp/modals/publish_exercise.html.haml

```diff
@@ -8,8 +8,9 @@
         By publishing your code, you&apos;ll help others learn from your work.
         You can choose which iterations you publish, add more iterations once it's published, and unpublish it at any time.
 
-      / TODO: There's probably a11y stuff to add to this tag
-      / If there's not, it can be deleted
+      -#
+        TODO: There's probably a11y stuff to add to this tag
+        If there's not, it can be deleted
```

Comment:
This is following this lint suggestion: https://github.com/sds/haml-lint/blob/main/lib/haml_lint/linter/README.md#consecutivecomments